src/netius/test/pool/common.py

```
@@ -28,16 +28,231 @@
+
+        # every event that was queued comes back in the order it was pushed,
+        # and the queue is left empty behind them
+        self.assertEqual(events, [("first", 1), ("second", 2)])
+        self.assertEqual(pool.events, [])
+
+        # a pop over an empty queue gives nothing back and asks nothing of
+        # the notification, as there is nothing to be undone
+        self.assertEqual(pool.pop_all(denotify=True), [])
+
+    def test_pop_all_denotify(self):
+        if mock == None:
```

Ruff (0.16.3)

[error] Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)